lib/search.js

```diff
@@ -148,15 +168,22 @@ module.exports = new class LibSearch {
                 score: 0
             }
 
+            // imdb, max score 7
+            if (input.imdbid && sub.imdbid === input.imdbid)
+                tmp.score += 5
+            if (input.season && input.episode && sub.season === input.season && sub.episode === input.episode)
+                tmp.score += 2
 
-            // score calculations
-            sub.MatchedBy === 'moviehash' && (tmp.score += 8)
-            sub.MatchedBy === 'tag' && (tmp.score += 7) || (tmp.score += matchTags(sub, 7))
-
-            let matchByFPS = sub.MovieFPS && input.fps && parseInt(sub.MovieFPS) > 0 && (sub.MovieFPS.startsWith(input.fps) || input.fps.toString().startsWith(sub.MovieFPS))
-            sub.MatchedBy === 'imdbid' && (tmp.score += 5) && matchByFPS && (tmp.score += 0.5)
-            !sub.MatchedBy.match(/moviehash|tag|imdbid/) && matchByFPS && (tmp.score += 2)
+            // version, max score 8
+            if (input.hash && sub.hash === input.hash || input.filesize && input.filesize === sub.filesize) {

```

Comment:
I have an issue with this, as something matching tags+fps (7+1) will score as high as something matching by hash (8), which should not be true, i would remove the if/else